GPU/Common/GPUCommonMath.h

```diff
@@ -367,9 +376,11 @@ GPUhdi() float GPUCommonMath::Copysign(float x, float y)
 #endif // GPUCA_GPUCODE
 }
 
-#ifndef GPUCA_GPUCODE
+#if !defined(GPUCA_GPUCODE) && defined(__GNUC__) && !defined(__clang__)
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wunused-value" // GCC BUG in omp atomic capture gives false warning
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
 #endif
```

Comment:
Can't we do 1 push and later on 1 pop?

Reply:
ah yeah, totally. Didn't pay attention to how push works. Removed the second push.